nifi-frontend/src/main/frontend/apps/nifi/src/app/ui/common/controller-service/move-controller-service/move-controller-service.component.ts

```diff
@@ -0,0 +1,158 @@
+import { MatCheckboxModule } from '@angular/material/checkbox';
+import { MatButtonModule } from '@angular/material/button';
+import { AsyncPipe, NgTemplateOutlet } from '@angular/common';
+import { MatTabsModule } from '@angular/material/tabs';
+import { MatOptionModule } from '@angular/material/core';
+import { MatSelectModule } from '@angular/material/select';
+import { ControllerServiceApi } from '../controller-service-api/controller-service-api.component';
+import { ControllerServiceReferences } from '../controller-service-references/controller-service-references.component';
+import { NifiSpinnerDirective } from '../../spinner/nifi-spinner.directive';
+import { TextTip } from '../../tooltips/text-tip/text-tip.component';
+import { NifiTooltipDirective } from '../../tooltips/nifi-tooltip.directive';
+import { ControllerServiceState } from '../../../../state/contoller-service-state';
+import { Store } from '@ngrx/store';
+import {
+    setControllerService,
+    stopPollingControllerService,
+} from '../../../../state/contoller-service-state/controller-service-state.actions';
+import { selectControllerService } from '../../../../state/contoller-service-state/controller-service-state.selectors';
+import { CloseOnEscapeDialog } from '../../close-on-escape-dialog/close-on-escape-dialog.component';
+import { moveControllerService } from './../../../../pages/flow-designer/state/controller-services/controller-services.actions';
+import { SelectOption } from './../../../../state/shared/index';
+
+@Component({
+    selector: 'move-controller-service',
+    standalone: true,
+    templateUrl: './move-controller-service.component.html',
+    imports: [
+        ReactiveFormsModule,
+        MatDialogModule,
+        MatInputModule,
+        MatCheckboxModule,
+        MatButtonModule,
+        MatTabsModule,
+        MatOptionModule,
+        MatSelectModule,
+        ControllerServiceApi,
+        ControllerServiceReferences,
+        AsyncPipe,
+        NifiSpinnerDirective,
+        NifiTooltipDirective,
+        NgTemplateOutlet
+    ],
+    styleUrls: ['./move-controller-service.component.scss']
+})
+export class MoveControllerService extends CloseOnEscapeDialog {
+    @Input() goToReferencingComponent!: (component: ControllerServiceReferencingComponent) => void;
+    protected readonly TextTip = TextTip;
+    protected controllerServiceActionProcessGroups: SelectOption[] = [];
+
+    controllerService$ = this.store.select(selectControllerService);
+
+    moveControllerServiceForm: FormGroup;
+
+    @ViewChild('stepComplete') stepComplete!: TemplateRef<any>;
+    @ViewChild('stepError') stepError!: TemplateRef<any>;
+    @ViewChild('stepInProgress') stepInProgress!: TemplateRef<any>;
+    @ViewChild('stepNotStarted') stepNotStarted!: TemplateRef<any>;
+
+    constructor(
+        @Inject(MAT_DIALOG_DATA) public request: MoveControllerServiceDialogRequest,
+        private store: Store<ControllerServiceState>,
```

Comment:
```suggestion
        private store: Store<NiFiState>,
```

Reply:
Made this change. Now using NiFiState store.